pandas/core/computation/ops.py

```diff
@@ -45,6 +45,7 @@
 _unary_math_ops = (
     "sin",
     "cos",
+    "tan",
```

Comment:
Can you add a test for this?

Reply:
Just by adding "tan" to the list `_unary_math_ops` a standard test (which is applied to all these functions) is automatically added: https://github.com/pandas-dev/pandas/blob/454e2e1d9d7b118953ecfb4edc6f9fe7f5cb07b8/pandas/tests/computation/test_eval.py#L1615
Let me know if you meant anything more specific.